fares-data-build-tool/create-data
# feat: autogenerate returns from singles

## Description

Gives the user the ability to automatically create a return from two eligible singles for the same service, as long as they meet certain criteria.

## Testing instructions

Create two singles which use the same service, same passenger type (id), and have both outbound and inbound directions, and ensure they are not expired. Then find one of them, and click the generate button.
